### PR TITLE
Add 1.2.0 version

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

**Comment by gbalabasquer** [comment]
@brianmcmichael Could you please get the ipfs deployed?
Kovan: https://github.com/makerdao/mcd-changelog/blob/1.2.0/releases/kovan/1.2.0/contracts.json
Mainnet: https://github.com/makerdao/mcd-changelog/blob/1.2.0/releases/mainnet/1.2.0/contracts.json

File: releases/mainnet/1.2.0/abi/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<html>
+	<div>
+		<h2>
+			<strong>Mainnet Release 1.2.0 - ABIs</strong>
+		</h2>
+		<p>Download an <a href="/releases/mainnet/1.1.4/abi/mainnet_abi_1.2.0.zip" target="_blank">archive of all Mainnet 1.2.0 ABIs</a>.<p>
```
Comment (zdumitru) [inline releases/mainnet/1.2.0/abi/index.html:6]: Should be 1.2.0 in both places

**Review by zdumitru — CHANGES_REQUESTED** [review]
See the comment above.

File: releases/mainnet/1.2.0/abi/index.html (hunk 1 above)
Comment (gbalabasquer) [reply releases/mainnet/1.2.0/abi/index.html]: Fixed

**Review by kmbarry1 — APPROVED** [review]
LGTM

**Review by brianmcmichael — APPROVED** [review]
LGTM

**Review by zdumitru — APPROVED** [review]

**Pull request merged**